packages/office-ui-fabric-react/src/components/Checkbox/examples/Checkbox.Basic.Example.tsx

```diff
@@ -42,7 +42,9 @@ export class CheckboxBasicExample extends React.Component<{}, ICheckboxBasicExam
             onBlur: () => { console.log('Uncontrolled checkbox is blured'); }
           } }
           styles={ styles }
+          ariaDescribedBy={ 'descriptionID' }
         />
+        <label id='descriptionID' className='screenReaderOnly'>Uncontroller checkbox description</label>
```

Comment:
you need to generate a dynamic id here so multiple instances don't get the same id